features/keymap/ncl/layers-set_layers_to.feature

```diff
@@ -0,0 +1,38 @@
+Feature: Layers (Set Layers To)
+
+  The `K.layer_mod.set_active_layers_to` key allows setting the active layers.
```

Comment:
The other features refer to (and link to) equivalent functionality in QMK, ZMK. That'd be worth doing here.